requirements-web.txt

```diff
@@ -104,9 +106,9 @@ jinja2==3.1.2 \
     --hash=sha256:31351a702a408a9e7595a8fc6150fc3f43bb6bf7e319770cbc0db9df9437e852 \
     --hash=sha256:6088930bfe239f0e6710546ab9c19c9ef35e29792895fed6e6e31a023a182a61
     # via flask
-mako==1.2.2 \
-    --hash=sha256:3724869b363ba630a272a5f89f68c070352137b8fd1757650017b7e06fda163f \
-    --hash=sha256:8efcb8004681b5f71d09c983ad5a9e6f5c40601a6ec469148753292abc0da534
+mako==1.2.1 \
```

Comment:
Are the `requirements*.txt` changes intentional? A lot of dependencies got downgraded, as well as the python version used to generate the files